### ISSUE TITLE
Some questions about the paper.

### ISSUE THREAD

**Issue by zou-longkun** [SimingYan/HPNet#10]
Hi, thanks for the code, and the great work. But there are a lot of places in the paper that confuse me， hope you can clarify：

**Comment by SimingYan** [comment]
Yes. Your first understanding is correct. My previous explanation was wrong. I have edited it. m1=128 and m2=m3=..=m(2k+1)=1.
